### ISSUE TITLE
Artifactory .NET Core NuGet Push Selector Error

### ISSUE THREAD

**Issue by jorymorrison** [jfrog/artifactory-azure-devops-extension#182]
**Describe the bug**
When utilizing the  `Artifactory .NET Core` task and using the `NuGet Push` command, the `Deployment repository` field dropdown list is not populated.

I see the error displayed in the Azure Devops UI as follows:

```
Selector could not parse response.. Exception Message: Unexpected character while parsing path query: (
```

**To Reproduce**
Azure DevOps Server Version Dev17.M153.5
JFrog Artifactory Extension Version 1.10.0 
JFrog Artifactory Service Connection established

**Expected behavior**
I expect the list field to populate with target artifactory repositories. 

**Screenshots**
https://user-images.githubusercontent.com/1889865/89679765-bed1e000-d8bf-11ea-80d5-73b7c5184383.png

**Versions**
- Artifactory Azure DevOps extension version: 1.10
- Agent operating system: Ubuntu 18
- JFrog Artifactory version: 7.6.3

**Additional context**
The correct behavior is observed when using the `Artifactory NuGet` task. I have used this task to successfully push `.nupkg` to Artifactory.

Cursory research shows a possibly interesting difference in the bindings between the functioning NuGet task and the non functioning .NET task.

NuGet Push:
```
        {
            "endpointId": "$(artifactoryService)",
            "endpointURL": "{{endpoint.url}}/api/repositories",
            "target": "targetDeployRepo",
            "resultSelector": "jsonpath:$[?(@.type=='LOCAL' || @.type=='VIRTUAL')]",
            "resultTemplate": "{ \"Value\" : \"{{{key}}}\", \"DisplayValue\" : \"{{{key}}}\" }"
        }
```

.NET NuGet Push
```
            "endpointId": "$(artifactoryService)",
            "endpointURL": "{{endpoint.url}}/api/repositories",
            "target": "targetDeployRepo",
            "resultSelector": "jsonpath:$[?(@.packageType=='NuGet' && (@.type=='LOCAL' || @.type=='VIRTUAL'))]",
            "resultTemplate": "{ \"Value\" : \"{{{key}}}\", \"DisplayValue\" : \"{{{key}}}\" }"
```

**Comment by jorymorrison** [comment]
A manually constructed YAML task based on the task.json functions as expected.

```
- task: ArtifactoryDotnetCore@1
  name: ExperimentalPush
  inputs:
    command: 'push'
    artifactoryService: 'artifactory'
    targetDeployRepo: 'nuget'
    pathtoNupkg: '$(Build.ArtifactStagingDirectory)/*.nupkg'
```

**Comment by jorymorrison** [comment]
Changing the resultSelector to the following works (I have tested with a local patch release). I have not identified what the problem is with the current one. I suspect it comes down to what implementation of JsonPath is being used, which I assume may be NewtonSoft in this case?

`"resultSelector": "jsonpath:$[?(@.type=='LOCAL' || @.type=='VIRTUAL')]",`

**Comment by sophietait** [comment]
reproduced issue

**Comment by RobiNino** [comment]
Hi @jorymorrison , @sophietait ,
Thanks for reporting this issue and investing time in finding it's root.
We just released 1.10.2 that includes a fix, and we'd appreciate your feedback for it.

**Comment by eyalbe4** [comment]
@jorymorrison and @sophietait,
I'm closing this issue for now, but please let us know if you need further assistance with this.
